testplan/web_ui/testing/src/Toolbar/InteractiveButtons.js

```diff
@@ -46,4 +65,204 @@ const ResetButton = (props) => {
   }
 };
 
-export {ResetButton};
+
+const showHistory = (historyExporters) => {
+  if (Array.isArray(historyExporters)) {
```

Comment:
This can be written as:

if (Array.isArray(historyExporters) && historyExporters.length > 0)

to spare an indent